lib/domain/db/models/timeline_pin_layout.dart

```diff
@@ -1,29 +1,29 @@
 enum TimelinePinLayout {
```

Comment:
I don't know why Maps in timeline_pin.g.dart are generated with strings but could we generate them with integers? That way build runner will generate `TimelinePinLayout => int` mapping for us and we can stop maintaining this huge switch case. Just a thought...

Reply:
These numbers must be constant since they correspond to integers that get sent to the watch. I don't think generating them is a good idea since generator can change values.